### PR TITLE
FEM: python code formatting

### PR DESCRIPTION
**Pull request by berndhahnebach**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, we ask you to conform to the following items. Pull requests which don't satisfy all the items below might be rejected. If you are in doubt with any of the items below, don't hesitate to ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10)!

- [x]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [x]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [x]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [x]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [x]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [x]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`

---

code formatting according https://github.com/FreeCAD/FreeCAD/blob/master/src/Mod/Fem/coding_conventions.md

### CODE REVIEW

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7161">pipeline status</a> for feature branch [PR_7161](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7161). Pipeline [584649816](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/584649816) was triggered at [70143d7](https://github.com/FreeCAD/FreeCAD/pull/7161/commits/70143d7dddb292e1a542ad01e6f7d40470267529). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7161">pipeline status</a> for feature branch [PR_7161](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7161). Pipeline [584652183](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/584652183) was triggered at [533d849](https://github.com/FreeCAD/FreeCAD/pull/7161/commits/533d8494867ba23a1828063a7c20fe735dcf9935). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by berndhahnebach** [comment]
@donovaly: any comments on this?

File: src/Mod/Fem/femobjects/constraint_electrostaticpotential.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 #  \ingroup FEM
 #  \brief constraint electrostatic potential object
 
-from FreeCAD import Units
+# from FreeCAD import Units
```
Comment (benj5378) [inline src/Mod/Fem/femobjects/constraint_electrostaticpotential.py:33]: Maybe just remove completely?

File: src/Mod/Fem/femtaskpanels/task_constraint_electrostaticpotential.py [hunk 2]
```diff
@@ -111,7 +112,7 @@ def _initParamWidget(self):
             not self._paramWidget.capacitanceBodyBox.isChecked())
 
     def _applyWidgetChanges(self):
-        unit = "V"
+        # unit = "V"
```
Comment (benj5378) [inline src/Mod/Fem/femtaskpanels/task_constraint_electrostaticpotential.py:115]: Remove completely?

File: src/Mod/Fem/femtaskpanels/task_result_mechanical.py [hunk 3]
```diff
@@ -46,7 +46,7 @@
 
 import FreeCAD
 import FreeCADGui
-import FemGui
+# import FemGui
```
Comment (benj5378) [inline src/Mod/Fem/femtaskpanels/task_result_mechanical.py:49]: Remove completely?

File: src/Mod/Fem/femtaskpanels/task_solver_ccxtools.py [hunk 4]
```diff
@@ -248,11 +248,11 @@ def calculixStarted(self):
...
     def calculixStateChanged(self, newState):
         if (newState == QtCore.QProcess.ProcessState.Starting):
-                self.femConsoleMessage("Starting CalculiX...")
+            self.femConsoleMessage("Starting CalculiX...")
         if (newState == QtCore.QProcess.ProcessState.Running):
-                self.femConsoleMessage("CalculiX is running...")
+            self.femConsoleMessage("CalculiX is running...")
         if (newState == QtCore.QProcess.ProcessState.NotRunning):
-                self.femConsoleMessage("CalculiX stopped.")
+            self.femConsoleMessage("CalculiX stopped.")
```
Comment (benj5378) [inline src/Mod/Fem/femtaskpanels/task_solver_ccxtools.py:255]: Bonus points for converting this to `elif`s

File: src/Mod/Fem/femviewprovider/view_mesh_gmsh.py [hunk 5]
```diff
@@ -206,12 +206,20 @@ def onDelete(self, feature, subelements):
         children = self.claimChildren()
         if len(children) > 0:
             # issue a warning
-            bodyMessage = "The mesh contains submesh objects, therefore the\nfollowing referencing objects might be lost:\n"
+            bodyMessage = (
```
Comment (benj5378) [inline src/Mod/Fem/femviewprovider/view_mesh_gmsh.py:209]: Every time I see this, I always read it as body massage the first time

File: src/Mod/Fem/femviewprovider/view_result_mechanical.py [hunk 6]
```diff
@@ -30,7 +30,7 @@
 #  \ingroup FEM
 #  \brief view provider for mechanical ResultObjectPython
 
-import FreeCAD
+# import FreeCAD
```
Comment (benj5378) [inline src/Mod/Fem/femviewprovider/view_result_mechanical.py:33]: Remove completely?

**Review by benj5378 — APPROVED** [review]

File: src/Mod/Fem/feminout/importZ88O2Results.py [hunk 7]
```diff
@@ -68,8 +68,8 @@ def insert(
 # ********* module specific methods *********
 def import_z88_disp(
     filename,
-    analysis = None,
-    result_name_prefix = None
+    analysis=None,
```
Comment (donovaly) [inline src/Mod/Fem/feminout/importZ88O2Results.py:71]: Why this change? usually we use spaces around binary operators.

File: src/Mod/Fem/femtaskpanels/task_constraint_electrostaticpotential.py (hunk 2 above)
Comment (donovaly) [inline src/Mod/Fem/femtaskpanels/task_constraint_electrostaticpotential.py:115]: Why do you remove this?

File: src/Mod/Fem/femobjects/constraint_electrostaticpotential.py (hunk 1 above)
Comment (donovaly) [inline src/Mod/Fem/femobjects/constraint_electrostaticpotential.py:33]: This seems wrong. App::PropertyElectricPotential uses a unit as far as I see at a quick look.

File: src/Mod/Fem/femtaskpanels/task_constraint_electrostaticpotential.py (hunk 2 above)
Comment (donovaly) [reply src/Mod/Fem/femtaskpanels/task_constraint_electrostaticpotential.py:115]: Yes, it no longer necessary, better remove it completely.

**Comment by donovaly** [comment]
> @donovaly: any comments on this?

Yes, see above.

Note that I started to use proper quantity spin boxes: https://github.com/FreeCAD/FreeCAD/commit/30928c07
They provide a proper unit handling because the user cannot change them. App::PropertyElectricPotential uses Voltage and this must not be changed.

File: src/Mod/Fem/femobjects/constraint_electrostaticpotential.py (hunk 1 above)
Comment (benj5378) [reply src/Mod/Fem/femobjects/constraint_electrostaticpotential.py:33]: But we don't use `App::PropertyElectricPotential` in the script?

File: src/Mod/Fem/femtaskpanels/task_result_mechanical.py (hunk 3 above)
Comment (berndhahnebach) [reply src/Mod/Fem/femtaskpanels/task_result_mechanical.py:49]: ok

File: src/Mod/Fem/feminout/importZ88O2Results.py (hunk 7 above)
Comment (berndhahnebach) [reply src/Mod/Fem/feminout/importZ88O2Results.py:71]: yes,  but not inside  the method parameter

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7161">pipeline status</a> for feature branch [PR_7161](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7161). Pipeline [585880354](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/585880354) was triggered at [001fb5d](https://github.com/FreeCAD/FreeCAD/pull/7161/commits/001fb5d166f5a5e91a68568dce4ab1d3512580f7). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by berndhahnebach** [comment]
> Note that I started to use quantity spin boxes: [30928c07](https://github.com/FreeCAD/FreeCAD/commit/30928c07)
> They provide a proper unit handling because the user cannot change them. App::PropertyElectricPotential uses Voltage and this must not be changed.


OK, but not in all FEM input fields? There are for sure use cases when it makes sense to change the unit.

**Comment by berndhahnebach** [comment]
> > @donovaly: any comments on this?
> 
 [30928c07](https://github.com/FreeCAD/FreeCAD/commit/30928c07) in this commit a object property was changed but it was not handled what happens on load of old files.

File: src/Mod/Fem/femviewprovider/view_result_mechanical.py (hunk 6 above)
Comment (berndhahnebach) [reply src/Mod/Fem/femviewprovider/view_result_mechanical.py:33]: ok

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7161">pipeline status</a> for feature branch [PR_7161](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7161). Pipeline [585896900](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/585896900) was triggered at [84c70b8](https://github.com/FreeCAD/FreeCAD/pull/7161/commits/84c70b8ce23166b065e5e80c07931e0c527615d0). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by berndhahnebach** [comment]
merged
